applications/FluidDynamicsApplication/custom_elements/qs_vms_dem_coupled.h

```diff
@@ -266,6 +282,26 @@ class QSVMSDEMCoupled : public QSVMS<TElementData>
         const TElementData& rData,
         double &rMassRHS) const override;
 
+    virtual Matrix GetAtCoordinate(
+        const typename TElementData::NodalTensorData& rValues,
+        const typename TElementData::ShapeFunctionsType& rN) const;
+
+    virtual double GetAtCoordinate(
+        const typename TElementData::NodalScalarData& rValues,
+        const typename TElementData::ShapeFunctionsType& rN) const override;
+
+    virtual array_1d<double, 3> GetAtCoordinate(
+        const typename TElementData::NodalVectorData& rValues,
+        const typename TElementData::ShapeFunctionsType& rN) const override;
+
+    virtual double GetAtCoordinate(
+        const double Value,
+        const typename TElementData::ShapeFunctionsType& rN) const override;
+
+    void InverseMatrix(
+        Matrix& r_matrix,
+        Matrix& r_inv_matrix);
```

Comment:
```suggestion
        Matrix& rMatrix,
        Matrix& rInvMatrix);
```
Camelcase in function signatures.